### ISSUE TITLE
Include suggestion for making keyboard accessible tooltips in docs

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1137]
We should add clarifying documentation to use `TooltipTabstop` in the `Tooltip` documentation if the element the tooltip is targeting is non-interactive and does not have keyboard access by default. This is common for "help" tooltips that may only display via some text or an icon.
